pyproject.toml

```diff
@@ -45,20 +46,24 @@ repository = "https://github.com/NVIDIA/NeMo-Guardrails"
 nemoguardrails = "nemoguardrails.__main__:app"
 
 [tool.poetry.dependencies]
-python = ">=3.9,!=3.9.7,<3.13"
+python = ">=3.9,!=3.9.7,<3.14"
 aiohttp = ">=3.10.11"
 annoy = ">=1.17.3"
 fastapi = ">=0.103.0,"
-fastembed = ">=0.2.2, <0.4.1"
+fastembed = [
+  { version = ">=0.2.2, <0.5", python = ">=3.9,<3.10" },
+  { version = ">=0.5.0, <=0.6.0", python = ">=3.10, <3.14" },
```

Comment:
Cannot we support the other fastembed versions?
`{ version = ">=0.2.2, <=0.6.0", python = ">=3.10, <3.14" },`